components/service/nimbus/metrics.yaml

```diff
@@ -0,0 +1,105 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# This file defines the metrics that are recorded by glean telemetry. They are
+# automatically converted to Kotlin code at build time using the `glean_parser`
+# PyPI package.
+---
+
+$schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
+
+nimbus_events:
+  enrollment:
+    type: event
+    description: >
+      Recorded when a user has met the conditions and is first bucketed into an
+      experiment (i.e. targeting matched and they were randomized into a bucket
+      and branch of the experiment). Expected a maximum of once per experiment
+      per user.
+    extra_keys:
+      experiment:
+        description: The slug/unique identifier of the experiment
+      branch:
+        description: The branch slug/identifier that was randomly chosen
+      enrollment_id:
+        description: A unique identifier generated at enrollment time
+    bugs:
+      - https://jira.mozilla.com/browse/SDK-61
+    data_reviews:
+      - TBD
```

Comment:
nit: let's remember to update this with details of the data review before landing :-)